akka-actor-typed/src/main/scala/akka/actor/typed/delivery/internal/ConsumerControllerImpl.scala

```diff
@@ -207,11 +220,44 @@ import akka.annotation.InternalApi
     if (ref.path.address.hasGlobalScope)
       throw new IllegalArgumentException(s"Consumer [$ref] should be local.")
   }
+
+  private class RetryTimer(
+      timers: TimerScheduler[ConsumerControllerImpl.InternalCommand],
+      minBackoff: FiniteDuration,
+      maxBackoff: FiniteDuration) {
+    private var interval = minBackoff
+
+    def start(): Unit = {
+      interval = minBackoff
+      timers.startTimerWithFixedDelay(Retry, interval)
+    }
+
+    def scheduleNext(): Unit = {
+      val newInterval =
+        if (interval eq maxBackoff)
+          maxBackoff
+        else
+          maxBackoff.min(interval * 1.5) match {
```

Comment:
1.5?

Reply:
that's also exponential backoff but less steep than 2